docs/c-runtime-library/reference/setmbcp.md

```diff
@@ -34,16 +34,6 @@ Returns 0 if the code page is set successfully. If an invalid code page value is
 
 The **_setmbcp** function specifies a new multibyte code page. By default, the run-time system automatically sets the multibyte code page to the system-default ANSI code page. The multibyte code page setting affects all multibyte routines that are not locale dependent. However, it is possible to instruct **_setmbcp** to use the code page defined for the current locale (see the following list of manifest constants and associated behavior results). For a list of the multibyte routines that are dependent on the locale code page rather than the multibyte code page, see [Interpretation of Multibyte-Character Sequences](../../c-runtime-library/interpretation-of-multibyte-character-sequences.md).
 
-The multibyte code page also affects multibyte-character processing by the following run-time library routines:
```

Comment:
Is this deletion because the behavior changed at some point, and if so, when? (I assume this was useful info once upon a time, but I acknowledge the danger of assumptions.)

Reply:
I believe this was still in affect in msvcr120.dll and changed with the first version of the UCRT.